src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/expressions.test.ts

```diff
@@ -29,14 +29,17 @@ describe('buildPartialMatcher', () => {
 
 describe('getExpressionType', () => {
   const getASTForExpression = (expression: string) => {
-    const { root } = parse(`FROM index | EVAL ${expression}`);
+    const { root, errors } = Parser.parse(`FROM index | EVAL ${expression}`);
+
+    if (errors.length > 0) {
+      throw new Error(
+        `Failed to parse expression "${expression}": ${errors.map((e) => e.message).join(', ')}`
+      );
+    }
+
     return root.commands[1].args[0];
   };
 
-  test('empty expression', () => {
-    expect(getExpressionType(getASTForExpression(''))).toBe('unknown');
-  });
```

Comment:
Why this removal?

Reply:
I'll check tomorrow. Something to do with the parsing behavior change. The `getExpressionType` returns `unknown` in case the node cast to boolean is false `if (!node)`. Maybe now it fails already at parsing, will check tomorrow.